netket/_vmc.py

```diff
@@ -1,17 +1,100 @@
+import math
 import sys
+import warnings
 
 import numpy as _np
-import math
 
 import netket as _nk
 from netket._core import deprecated
 from .operator import local_values as _local_values
 from netket.stats import statistics as _statistics, mean as _mean
 
-from netket.vmc_common import info
+from netket.vmc_common import info, tree_map
 from netket.abstract_variational_driver import AbstractVariationalDriver
 
 
+# Higher-level VMC functions:
+def estimate_expectations(
+    ops,
+    sampler,
+    n_samples,
+    n_discard=None,
+    compute_gradients=None,
+    return_gradients=False,
+    return_intermediate=False,
+):
+    """
+    For a pytree of linear operators, computes a statistical estimate of the
+    respective expectation values, variances, and optionally gradients of the
+    expectation values with respect to the variational parameters.
```

Comment:
If I'm not mistaken our estimate of the gradient assumes the operator be hermitian.
For the sake of being correct... could you write this remark?

Reply:
Ah, I did not think about that. I'll check and, if that is the case, add the remark.